prow/pubsub/subscriber/metrics.go

```diff
@@ -36,7 +39,7 @@ var (
 	errorCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Name: "prow_pubsub_error_counter",
 		Help: "A counter of the webhooks made to prow.",
-	}, []string{subscriptionLabel})
+	}, []string{subscriptionLabel, errorTypeLabel})
```

Comment:
Do you know what will happen to the existing metric data that doesn't have this label?

Reply:
I think I've seen it before, it's default to empty string when this field doesn't exist